Node/core/lib/locale/es/BotBuilder.json

```diff
@@ -1,16 +1,16 @@
 {
     "default_text": "No se pudo reconocer su respuesta. Por favor vuelva a intentar.",
     "default_number": "No se pudo reconocer el texto como número. Vuelva a intentar sólo con números.",
-    "default_confirm": "No se pudo reconocer la confirmación. Por favor responda con 'si' o 'no'.",
+    "default_confirm": "No se pudo reconocer la confirmación. Por favor responda con 'sí' o 'no'.",
     "default_choice": "No se pudo reconocer su respuesta. Por favor seleccione una opción de la lista.",
     "default_time": "No se pudo reconocer el formato de la fecha. Por favor vuelva a intentar (MM/DD/YYYY HH:MM:SS).",
     "default_file": "No se pudo recibir el archivo. Por favor vuelva a intentar.",
     "default_error": "Oops. Ocurrió un problema. Por favor vuelva a intentar.",
     "list_or": " o ",
     "list_or_more": ", o ",
-    "confirm_yes": "si",
+    "confirm_yes": "sí",
     "confirm_no": "no",    
-    "boolean_choices": "true=y,yes,s,si,vale,ok,\u1F44D,\u1F44C|false=n,no,falso,\u1F44E,\u270B,\u1F590",
-    "yesExp": "^(1|s|si|vale|ok)(\\W|$)",
+    "boolean_choices": "true=y,yes,s,sí,vale,ok,\u1F44D,\u1F44C|false=n,no,falso,\u1F44E,\u270B,\u1F590",
+    "yesExp": "^(1|s|sí|vale|ok)(\\W|$)",
```

Comment:
It would be nice that both `sí` and `si` were accepted in `boolean_choices` and `yesExp`, as some people will not write the accent when typing.